### ISSUE TITLE
Spike: Identify usages of view components

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#19]
### Expected result

Which view/controller or view/manager combos are suitable for a rewrite to view components?

### Additional context

ASP.NET Core MVC view components are views that basically:

* don't use model binding
* are accompanied by a piece of imperative code

### Resources

* https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-components?view=aspnetcore-3.1

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #27
